exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatchSizer.java

```diff
@@ -199,12 +422,18 @@ public String toString() {
            .append(", per-array: ")
            .append(estElementCountPerArray);
       }
-      buf .append(", std size: ")
-          .append(stdSize)
-          .append(", actual size: ")
-          .append(estSize)
-          .append(", data size: ")
-          .append(dataSize)
+      buf .append(", std size per entry: ")
+          .append(getStdDataSizePerEntry())
+          .append(", std net size per entry: ")
+          .append(getStdNetSizePerEntry())
+          .append(", data size per entry: ")
+          .append(getDataSizePerEntry())
+          .append(", net size per entry: ")
+          .append(getNetSizePerEntry())
+          .append(", totalDataSize: ")
+          .append(getTotalDataSize())
+          .append(", totalNetSize: ")
+          .append(getTotalNetSize())
```

Comment:
Extra info is always good. This info is printed by the sort operator when debug logging is enabled. (It was invaluable when investigating issues found by QA tests.) Suggestion: find a way to compact the labels. Maybe:
```
Per entry: std size: xx, std net: xxx; Totals: data size: xxx, net size: xxx
```

Also, `totalDataSize` and `totalNetSize` should have spaces: they are labels, not variable names here.